test/ActiveLogin.Authentication.BankId.AspNetCore.Test/Helpers/TokenHelper.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Text.RegularExpressions;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.Test.Helpers
+{
+    public static class TokenHelper
```

Comment:
Small thing. Would propose naming it to `TokenExtractor` and make it internal.

Reply:
I love small things :D